### PR TITLE
Postgres end-to-end tests - Fix VulnScanWithGraphQLTest GraphQL queries

### PR DESCRIPTION
**Pull request by rhybrillou**
## Description

With the re-design of the dackbox graph for the postgres migration, some graphQL queries in the end-to-end tests need to be adapted.
Here, the focus is on the VulnScanWithGraphQLTest.groovy test file.

In order to make debugging easier, additional logs were added to the GraphQL service, and a minor framework adjustment was also introduced (add a function to tell whether the current run is a Postgres run or not).

## Checklist
- [ ] Investigated and inspected CI test results
~~- [ ] Unit test and regression tests added~~
~~- [ ] Evaluated and added CHANGELOG entry if required~~
~~- [ ] Determined and documented upgrade steps~~
~~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~~

If any of these don't apply, please comment below.

## Testing Performed

Test code fixes, CI run should be sufficient.

### CODE REVIEW

File: qa-tests-backend/src/test/groovy/BaseSpecification.groovy [hunk 1]
```diff
@@ -382,6 +382,10 @@ class BaseSpecification extends Specification {
         orchestrator.deleteSecret("gcr-image-pull-secret", Constants.ORCHESTRATOR_NAMESPACE)
     }
 
+    static Boolean isPostgresRun() {
```
Comment (janisz) [inline qa-tests-backend/src/test/groovy/BaseSpecification.groovy:385]: ❤️ 
Could you replace all `Env.CI_JOBNAME.contains("postgres")` with this method.

Comment (janisz) [reply qa-tests-backend/src/test/groovy/BaseSpecification.groovy:385]: Could be in another PR

**Review by janisz — APPROVED** [review]

File: qa-tests-backend/src/test/groovy/BaseSpecification.groovy (hunk 1 above)
Comment (rhybrillou) [reply qa-tests-backend/src/test/groovy/BaseSpecification.groovy:385]: Will do in a separate PR.

**Pull request merged**